### ISSUE TITLE
Devfiles in the OpenShift Dev Console samples are invalid

### ISSUE THREAD

**Issue by l0rd** [devfile/api#666]
/kind bug
/area registry

## Bug Summary

These 4 OpenShift console samples have a invalid devfile.yaml at their root ([1](https://github.com/redhat-developer/devfile-sample/blob/master/devfile.yaml), [2](https://github.com/elsony/devfile-sample-java-springboot-basic/blob/main/devfile.yaml), [3](https://github.com/elsony/devfile-sample-python-basic/blob/main/devfile.yaml), [4](
https://github.com/elsony/devfile-sample-code-with-quarkus/blob/main/devfile.yaml)):

![image](https://user-images.githubusercontent.com/606959/140045242-1de3b9e7-c240-4958-a72b-8dbde71ca2e7.png)

All have a `buildguidance` container referencing a nonexistent `buildguidanceimage-placeholder`:

```
  - name: buildguidance
    attributes:
      tool: console-import
    container:
      image: buildguidanceimage-placeholder. # <== this image doesn't exist
``` 

As a consequence CRW fails to start the workspaces ([related issue](https://issues.redhat.com/browse/CRW-2411)).

## Additional context

https://github.com/openshift/console/issues/8729

**Comment by l0rd** [comment]
This issue should be fixed when those devfiles will be replaced by valid v2.2.0 devfiles. @yangcao77 @elsony please share an example of such a file so that we can test on our hand if those will work with CRW 2.13.

**Comment by l0rd** [comment]
There are 3 pull requests to update the devfiles to 2.2 but those won't fix the samples currently included in ODC, they fix devfiles that will eventually be included in future versions of ODC:

https://github.com/devfile-samples/devfile-sample-java-springboot-basic/pull/1
https://github.com/devfile-samples/devfile-sample-code-with-quarkus/pull/1
https://github.com/devfile-samples/devfile-sample-python-basic/pull/1

**Comment by yangcao77** [comment]
@l0rd The change is now in ODC 4.9 and 4.10, to consume devfile 2.2.0 samples from registry. Are we good to close this issue?

**Comment by l0rd** [comment]
@yangcao77 sure, closing this issue. 
Can you please help me to find the developer console PRs that address the problem (change the git repos)? 
Do you know what version of OCP 4.9 include the fix? I have deployed 4.9.12 this morning but the sample git repos are the old one.

**Comment by yangcao77** [comment]
@l0rd 
4.10: https://github.com/openshift/console/pull/10394
4.9: https://github.com/openshift/console/pull/10620

Just noticed that it's not merged into 4.9 yet. The PR has been tested and approved, but requires console team to help merge it. I will track on that PR and let you know once it's merged.
